web-client/src/presenter/actions/CaseDetail/editRemoteStatusAction.ts

```diff
@@ -0,0 +1,41 @@
+import { state } from '@web-client/presenter/app.cerebral';
+import { Case } from '@shared/business/entities/cases/Case';
+
+/**
+ * calls the editRemoteStatusInteractor to indicate remote status
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {Function} providers.get the cerebral get function
+ * @returns {object} the alertSuccess and updated caseDetail object
+ */
+export const editRemoteStatusAction = async ({
+  applicationContext,
+  get,
+}: ActionProps) => {
+  const docketNumber = get(state.caseDetail.docketNumber);
+  const modalState = get(state.modal);
+  const { remoteTrialGrantedDate } = modalState;
+  const rawCase = get(state.caseDetail);
```

Comment:
Could cut out a const by directly deconstructing the modal state thats returned from this get call. Not important, but it might help readability on this variable block.

```suggestion
  const docketNumber = get(state.caseDetail.docketNumber);
  const { remoteTrialGrantedDate } = get(state.modal);
  const rawCase = get(state.caseDetail);
```

Reply:
For sure, I think it reads much better now. Thank you!